src/discussions/comments/reply/Reply.jsx

```diff
@@ -26,22 +64,18 @@ function Reply({
   }, [reply.id]);
 
   return (
-    <div className="discussion-reply d-flex flex-column mt-2">
-      <div className="d-flex flex-row">
-        <div className="d-flex flex-column flex-fill">
-          <div className="status small">
-            <span className="font-weight-bold text-info-300 mr-1">
-              {reply.author}
-            </span>
-            <span title={reply.createdAt}>
-              {intl.formatMessage(messages.replyTime, {
-                relativeTime: timeago.format(reply.createdAt, intl.locale),
-              })}
-            </span>
-          </div>
-          <div className="reply-body d-flex" dangerouslySetInnerHTML={{ __html: reply.renderedBody }} />
-        </div>
-        <CommentIcons abuseFlagged={reply.abuseFlagged} following={reply.following} />
+    <div className="bt-1 discussion-reply d-flex flex-column my-2">
+      <div>
+        <ReplyHeader reply={reply} intl={intl} />
+        <div className="reply-body d-flex px-2" dangerouslySetInnerHTML={{ __html: reply.renderedBody }} />
```

Comment:
why it is safe to use the dangerous method.? Can we make sure that It will not create security issues like `XXS attacks`?

Reply:
@awais-ansari Any time you are putting user-generated HTML in a page there is some risk, which is why the only way to include html inside react is to use this attribute with a scary name (you can read more on this [here](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml) )

However, in this case, we have a few mitigations:
- This HTML is *not* user generated. The users enter markdown and that is first rendered to HTML, and the resulting HTML is stripped of unsafe code [here](https://github.com/edx/edx-platform/blob/29f29f9da1db2bc3764fc3c4eb911c6d9e0c1e96/lms/djangoapps/discussion/rest_api/render.py#L91). 
- Since the HTML is already sanitised, it is relatively safe to inject in the browser this way. 
- Additionally,  this is temporary, the future plan is to replace this with a markdown renderer in browser, but till then to have a usable, testable UI we are using this.